core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,105 @@
+            <h2><label class="oppia-topic-name-text">{{ topicName }}</label></h2>
+          </mat-card>
+          <div *ngIf="!checkIfEmpty(skills)" *ngFor="(subtopicName, skills) in subtopics" class="list-view-item ">
+            <mat-card class="list-view-item">
+              <label class="oppia-subtopic-name-text">{{ subtopicName }}</label>
+              <md-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="skill in skills | filter:skillFilterText" ng-value="skill.id" aria-label="{{ skill.description }}">
+                {{ skill.description }}
+              </md-radio-button>
+            </mat-card>
+          </div>
+        </mat-card>
+        <mat-card *ngIf="untriagedSkillSummaries.length" class="list-view-item">
+          <mat-card layout="row" class="list-view-item">
+            <h2><label class="oppia-heading-text">Untriaged Skills</label></h2>
+          </mat-card>
+          <div *ngIf="!checkIfEmpty(untriagedSkillSummaries)" class="list-view-item ">
+            <mat-card class="list-view-item">
+              <md-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="skill in untriagedSkillSummaries | filter:skillFilterText" ng-value="skill.id" aria-label="{{ skill.description }}">
+                {{ skill.description }}
+              </md-radio-button>
+            </mat-card>
+          </div>
+        </mat-card>
+      </mat-radio-group>
+    </div>
+  </div>
+  <div class="list-group oppia-filter-list-group">
+    <mat-card *ngIf="canAllowSkillsFromOtherTopics()" layout="row" class="list-view-item oppia-list-view-item">
+      <label>Filter By Topics</label>
+    </mat-card>
+    <div *ngIf="canAllowSkillsFromOtherTopics()" class="oppia-list-view-item-container">
+      <mat-card class="list-view-item ">
+        <md-checkbox class="oppia-skills-list-item" *ngFor="topic in topicFilterList" ng-model="topic.checked" ng-change="updateSkillsListOnTopicFilterChange()">{{ topic.topicName }}</md-checkbox>
+      </mat-card>
+    </div>
+    <br>
+    <mat-card layout="row" class="list-view-item oppia-list-view-item">
+      <label>Filter By Subtopic</label>
+    </mat-card>
+    <div class="oppia-list-view-item-container">
+      <mat-card class="list-view-item">
+        <div *ngFor="(topicName, subtopics) in subTopicFilterDict">
+          <h3><label class="list-group-item-heading oppia-heading-text">{{ topicName }}</label></h3>
+          <mat-card class="list-view-item">
+            <md-checkbox class="oppia-skills-list-item" *ngFor="subtopic in subtopics" ng-model="subtopic.checked" ng-change="updateSkillsListOnSubtopicFilterChange()">{{ subtopic.subTopicName }}</md-checkbox>
+          </mat-card>
+        </div>
+      </mat-card>
+    </div>
+    <br>
+    <button class="btn btn-secondary" (click)="clearAllFilters()">Clear All Filters</button>
+  </div>
+</div>
+<style>
+  skill-selector .skill-selector-parent,
```

Comment:
In general, during migration, I have found styles selectors that use tags to not work when the tag is the selector of this component. If this is the case here too, you can wrap the entire template with a div of class `skill-selector` and change the CSS selector from `skill-selector .skill-selector-parent` to `.skill-selector .skill-selector-parent`. (Notice the extra dot at the starting).